### PR TITLE
[Synfig Studio] Add Eyedrop Tool legend in Toolbox

### LINKED ISSUE #1908: Add Eyedrop Tool keyboard accelerators legend in Toolbox
**Synfig version & platform**:
master 9b580c6

**Issue description**:
Now with `Eyedrop Tool` it's possible to:

- `Clic` to assign `Outline Color`
- `Ctrl + Clic` to assign `Fill Color`

and this option is not displayed in the `Tool Options` window.

I suggest to add a legend much like `Transform Tool`:

- Clic to assign Outline Color
- Ctrl + Clic to assign Fill Color

### PR DESCRIPTION
**Pull request by FirasH**
Fix #1908

eyedrop-tool

### CODE REVIEW

File: synfig-studio/src/gui/states/state_eyedrop.cpp [hunk 1]
```diff
@@ -104,16 +112,55 @@ StateEyedrop_Context::StateEyedrop_Context(CanvasView *canvasView):
...
+	title_label.set_label(_("Eyedrop Tool"));
+	Pango::AttrList list;
+	Pango::AttrInt attr = Pango::Attribute::create_attr_weight(Pango::WEIGHT_BOLD);
+	list.insert(attr);
+	title_label.set_attributes(list);
+	title_label.set_alignment(Gtk::ALIGN_START, Gtk::ALIGN_CENTER);
+
+	options_table.attach(title_label,
+		0, 2, 0, 1, Gtk::FILL, Gtk::FILL, 0, 0);
+	options_table.attach(*manage(new Gtk::Label(_("Clic to assign Outline Color"), Gtk::ALIGN_START)),
```
Comment (ice0) [inline synfig-studio/src/gui/states/state_eyedrop.cpp:125]: `Click`

File: synfig-studio/src/gui/states/state_eyedrop.cpp [hunk 2]
```diff
@@ -104,16 +112,55 @@ StateEyedrop_Context::StateEyedrop_Context(CanvasView *canvasView):
...
+		0, 2, 1, 2, Gtk::EXPAND|Gtk::FILL, Gtk::EXPAND|Gtk::FILL, 0, 0);
+	options_table.attach(*manage(new Gtk::Label(_("Ctrl + Clic to assign Fill Color"), Gtk::ALIGN_START)),
```
Comment (ice0) [inline synfig-studio/src/gui/states/state_eyedrop.cpp:127]: `Click` - same here

**Comment by FirasH** [comment]
@ice0 changed as you suggested

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**